create-database.sql

```diff
@@ -1,53 +1,50 @@
 CREATE DATABASE ShopDB;
+
 USE ShopDB;
 
+
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50) NOT NULL,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE Products (
     ID INT AUTO_INCREMENT,
-    Name VARCHAR(50),
+    Name VARCHAR(50) NOT NULL,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE Warehouses (
     ID INT AUTO_INCREMENT,
-    Name VARCHAR(50),
-    Address VARCHAR(50),
-    CountryID INT,
+    Name VARCHAR(50) NOT NULL,
+    Address VARCHAR(50) NOT NULL,
+    CountryID INT NOT NULL,
 	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE ProductInventory (
-    ID INT,
-    ProductID INT,
+    ID INT AUTO_INCREMENT,
```

Comment:
When inserting data into tables with `AUTO_INCREMENT` fields, such as `ProductInventory`, it's best practice to omit the ID values. The database will automatically handle these, ensuring unique and sequential IDs.